packages/blade/src/components/Tag/types.ts

```diff
@@ -0,0 +1,67 @@
+import type { StyledPropsBlade } from '~components/Box/styledProps';
+import type { IconComponent } from '~components/Icons';
+import type { StringChildrenType, TestID } from '~utils/types';
+
+type TagProps = {
+  /**
+   * Decides the size of Tag
+   *
+   * @default medium
+   */
+  size?: 'medium' | 'large';
+
+  /**
+   * Leading icon for your Tag
+   */
+  icon?: IconComponent;
+
+  /**
+   * Callback when close icon on Tag is clicked
+   */
+  onDismiss: () => void;
+
+  /**
+   * Text that renders inside Tag
+   */
+  children: StringChildrenType;
+
+  /**
+   * Disable tag
+   */
+  isDisabled?: boolean;
+
+  /**
+   * Private property for Blade.
+   *
+   * Should not be used by consumers.
+   *
+   * Used for adding virtual focus on tag.
+   *
+   * @private
+   */
+  _isVirtuallyFocussed?: boolean;
```

Comment:
Focused has only 1 's' 😅 
```suggestion
  _isVirtuallyFocused?: boolean;
```

Reply:
lol let me rename in that final docs PR otherwise it will create conflicts.